docs/errors.asciidoc

```diff
@@ -3,35 +3,11 @@
 
 An error record represents one error event, captured by Elastic APM agents within one service. 
 It is identified by a unique ID.
+Errors contain information about the `exception` that has been captured
+and some meta information about the setting in which the error occured. 
 
-Some of the key attributes of an error are described in more detail:
+Errors can either relate to monitored <<transactions, transactions>> 
+or be monitored stand alone, 
+depending on how your APM agents are set up.
```

Comment:
reading this, i guess someone will want to know how to set up an agent to relate errors and transactions...